src/llm_vm/completion/data_synthesis.py

```diff
@@ -1,20 +1,86 @@
 import json
 import sys 
+from sentence_transformers import SentenceTransformer, util
+
 
 class DataSynthesis:
```

Comment:
oh is your branch ontop of Viks? i thought that was merged right?

Reply:
it says I'm consistent with the main branch right now I think